tests/test_dict.py

```diff
@@ -88,19 +87,20 @@
 }
 
 
-def get_document(base_uri: str):
-    return TEST_DATA[base_uri]
+class TestResolveURI:
 
+    @classmethod
+    def setup_class(cls):
 
-@pytest.fixture(scope="module", autouse=True)
-def override_loader():
-    patcher = patch("json_ref_dict.ref_pointer.get_document", get_document)
-    mock_loader = patcher.start()
-    yield mock_loader
-    patcher.stop()
+        # pylint:disable=unused-variable
+        @loader.register
```

Comment:
I think it will be more DRY to keep this as a module-level fixture, but using your new functionality instead of `patch`. Saves repeating the `setup_class` on each test case.

E.g.
```python
@pytest.fixture(scope="module", autouse=True)
def override_loader():
    @loader.register
    def _get_document(base_uri: str):
        return TEST_DATA[base_uri]
    try:
        yield
    finally:
        loader.loaders.clear()
```

Reply:
done